records_mover/__init__.py

```diff
@@ -1,4 +1,13 @@
-from .types import JsonSchema  # noqa
```

Comment:
I'm not sure why this was here, but the tests and mypy are fine with me removing it, so probably no better time!